manifests/init.pp

```diff
@@ -208,10 +210,11 @@
     ssl_cert_password_file => $certs::qpid::nss_db_password_file,
   } ~>
   class { '::katello::qpid':
-    client_cert           => $certs::qpid::client_cert,
-    client_key            => $certs::qpid::client_key,
-    katello_user          => $user,
-    candlepin_event_queue => $candlepin_event_queue,
+    client_cert             => $certs::qpid::client_cert,
+    client_key              => $certs::qpid::client_key,
+    katello_user            => $user,
+    candlepin_event_queue   => $candlepin_event_queue,
+    candlepin_qpid_exchange => $::katello::candlepin_qpid_exchange,
```

Comment:
This is broken on Puppet 4 because nothing is actually loading the katello name space.  Best bet is to provide a value in katello_devel/params.pp I think